### PR TITLE
Fix import errors

### PR DESCRIPTION
**Pull request by rim99**

### CODE REVIEW

File: cheroot/test/helper.py [hunk 1]
```diff
@@ -40,7 +41,7 @@ class CherootWebCase(webtest.WebCase):
     scheme = 'http'
 
     available_servers = {
-        'wsgi': cheroot.server.WSGIServer,
+        'wsgi': cheroot.wsgi.WSGIServer,
```
Comment (webknjaz) [inline cheroot/test/helper.py]: I think it's `cheroot.wsgi.Server`

Comment (rim99) [inline cheroot/test/helper.py]: At the end of the `wsgi.py` file, `WSGIServer` class was defined as an alias of the `Server` class. So both are right.

```
# compatibility aliases
globals().update(
    WSGIServer=Server,
    WSGIGateway=Gateway,
    WSGIGateway_u0=Gateway_u0,
    WSGIGateway_10=Gateway_10,
    WSGIPathInfoDispatcher=PathInfoDispatcher,
)
```

Comment (webknjaz) [inline cheroot/test/helper.py]: Yes, but it seems those names are added for backwards compatibility and may be a subject of deprecation in future. So it's always better to use real names.

@jaraco am I missing anything?

Comment (rim99) [inline cheroot/test/helper.py]: Right, I get your point.

So should I just make a new commit on this one, or pull another request?

I think the latter is better?

Comment (webknjaz) [inline cheroot/test/helper.py]: When doing pull requests you may update it by pushing new commits into your branch or rewrite it with rebase and force push. There's no need to create new PR after each review :) 
Moreover, if you update commented lines, the related comments will automatically become collapsed

Comment (rim99) [inline cheroot/test/helper.py]: Yeah, I also feel weird for what I just did.

 I start to get familiar with GitHub quite recently. Thank you for all these hints. ^o^

**Pull request closed without merging**